assembl/static2/js/app/reducers/adminReducer/adminSections.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import { combineReducers } from 'redux';
+import type ReduxAction from 'redux';
+import { fromJS, List, Map } from 'immutable';
+import {
+  type Action,
+  UPDATE_SECTIONS,
+  UPDATE_SECTION_TITLE,
+  UPDATE_SECTION_URL,
+  TOGGLE_EXTERNAL_PAGE,
+  CREATE_SECTION,
+  DELETE_SECTION,
+  UP_SECTION,
+  DOWN_SECTION
+} from '../../actions/actionTypes';
+import { updateInLangstringEntries } from '../../utils/i18n';
+
+export const sectionsHaveChanged = (state: boolean = false, action: ReduxAction<Action>) => {
```

Comment:
missing type for return value (boolean). You can also create a type for the function, I find it more readable and you can use it before combineReducer to type the whole adminSections reducer. See examples here: https://github.com/assembl/assembl/commit/26b3ca97e4ff343a9867d7d7ed78686b27943fcb